code/stable/projectile/SRS/Projectile_SRS.tex

```diff
@@ -350,20 +352,20 @@ \subsubsection{General Definitions}
 \end{minipage}
 \paragraph{Detailed derivation of rectilinear velocity:}
 \label{GD:rectVelDeriv}
-Assume we have rectilinear motion of a particle (of negligible size and shape, from \hyperref[pointMass]{A: pointMass}); that is, motion in a straight line. The velocity is $v$ and the acceleration is $a$. The motion in \hyperref[TM:acceleration]{TM: acceleration} is now one-dimensional with a constant acceleration, represented by ${a^{c}}$. The initial velocity (at $t=0$, from \hyperref[timeStartZero]{A: timeStartZero}) is represented by ${v^{\text{i}}}$. From \hyperref[TM:acceleration]{TM: acceleration}, using the above symbols we have:
+Assume we have rectilinear motion of a particle (of negligible size and shape, from \hyperref[pointMass]{A: pointMass}); that is, motion in a straight line. The velocity is ${v^{\text{f}}}$ and the acceleration is $a$. The motion in \hyperref[TM:acceleration]{TM: acceleration} is now one-dimensional with a constant acceleration, represented by ${a^{c}}$. The initial velocity (at $t=0$, from \hyperref[timeStartZero]{A: timeStartZero}) is represented by ${v^{\text{i}}}$. From \hyperref[TM:acceleration]{TM: acceleration}, using the above symbols we have:
 
 \begin{displaymath}
-{a^{c}}=\frac{\,dv}{\,dt}
+{a^{c}}=\frac{\,d{v^{\text{f}}}}{\,dt}
 \end{displaymath}
 Rearranging and integrating, we have:
 
 \begin{displaymath}
-\int_{{v^{\text{i}}}}^{v}{1}\,dv=\int_{0}^{t}{{a^{c}}}\,dt
+\int_{{v^{\text{i}}}}^{{v^{\text{f}}}}{1}\,d{v^{\text{f}}}=\int_{0}^{t}{{a^{c}}}\,dt
```

Comment:
Here, it's even worse. I think the `t` is free, i.e. not the final time, so both the upper `v` and the one in the `dv` should be just `v`. Even if `t` was `t^f`, the `dv` part should still not have a superscript.